sdks/typescript/src/apache_beam/version.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require("fs");
+const path = require("path");
+
+// TODO: (Typescript) Is there a more standard way to do this?
+export const version: string = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "..", "..", "..", "package.json"))
+)["version"];
```

Comment:
```suggestion
export const { version } = require('../../../package.json');
```

Rather than relying on a runtime read (which doesn't get the benefits of the TS compiler checks), we can require import this.